src/app/components/site/Header.tsx

```diff
@@ -29,31 +27,23 @@ export const Header = () => {
               <div className="header-links ml-auto pr-3 px-md-3 d-flex align-items-center d-print-none">
                 {user &&
                   (!user.loggedIn ? (
-                    <React.Fragment>
-                      <div className="login mx-5 mx-sm-2">
-                        <Link to="/login">
-                          <span>Log in</span>
-                        </Link>
-                      </div>
-                      <div className="signup m-0 mr-md-4 ml-md-3">
-                        <Link to="/register">
-                          <span>Sign up</span>
-                        </Link>
-                      </div>
-                    </React.Fragment>
+                    <div className="login mx-5 mx-sm-2 d-flex flex-row">
```

Comment:
Can we look at the padding on this please compared to figma? the spacing between the text seems off, and the alignment on mobile doesn't match.

Reply:
I have used mx-4 to move it slightly to the right. 
A reason for it to not be more to the right is because we have. hamburger menu which is taking space, compared to Figma where there is not one. 
I have asked the design and Vicky and they are happy with it